docs/hugo/content/guide/asov1-asov2-migration/_index.md

```diff
@@ -147,13 +160,17 @@ spec:
 
 Once you have `resources.yaml` locally, examine it to ensure that it has the resources you expect.
 
-> ℹ️ **Note**: `asoctl` is importing the resources from Azure. This means it will not preserve any Kubernetes labels/annotations
-> you have on your ASOv1 resources. If your resources need certain labels or annotations, add those to `resources.yaml` 
-> manually or use the `--annotation` and `--label` arguments to `asoctl`. 
+{{% alert title="Note" %}}
+`asoctl` is importing the resources from Azure. This means it will not preserve any Kubernetes labels/annotations
```

Comment:
It's not a policy choice, but a limitation, because those annotations aren't present in Azure. Maybe we need to be more explicit about this?

```suggestion
`asoctl` is importing the resources from Azure. This means it cannot preserve any Kubernetes labels/annotations
```